src/trio/__init__.py

```diff
@@ -28,6 +28,7 @@
     MemoryReceiveChannel as MemoryReceiveChannel,
     MemorySendChannel as MemorySendChannel,
     open_memory_channel as open_memory_channel,
+    raise_single_exception_from_group as raise_single_exception_from_group,
```

Comment:
I don't think this should be publicly exported. I think service nurseries take make (if not all) of its use cases and I'd rather encourage their use whenever we get them.

Reply:
I feel like there might still be scenarios where you want this in more complex stuff going on, depending on exactly how the service-nursery implementation ends up - but we can leave it private for now and see how usage pans out.